README.md

```diff
@@ -1,43 +1,54 @@
-* The Handy
-* Vorze Piston
-
-(Note: all of these devices are onaholes.)
+Currently all linear devices covered by Buttplug are onaholes.
 
 ## Installation
 Prerequisites:
 * Install [Intiface Desktop](https://intiface.com/desktop/).
 * Install [BepInEx 5.3](https://github.com/BepInEx/BepInEx/releases) or later.
-* Install [KKAPI 1.10.0](https://github.com/IllusionMods/IllusionModdingAPI) or later.
 
-Download the DLLs from the latest release and move them into BepInEx\plugins under your game directory.
+**If you currently have v1.1.0 or v1.1.0 installed:** Delete ``Koikatsu Party\BepInEx\plugins\KK_ButtPlugin.dll`` before proceeding.
+
+Download **ButtPlugin.zip** from the latest release and drag the folder in the zip file into your Koikatsu installation folder. If you've done it right, there should be a folder called ``Koikatsu Party\BepInEx\plugins\KK_ButtPlugin`` with some DLL files inside.
 
 ## How to use
 1. Open Intiface Desktop.
 1. Start the server.
-1. Add the device you want to use.
+1. Add the device you want to use. You might have to pair it first.
 1. Disconnect Intiface from the server, but don't close it (this step is important).
 1. Start Koikatsu in either desktop or VR mode.
 1. You know the rest.
 
-All Insert animations should be accurately synced to the device. Some Receive animations will be out of sync because I haven't got around to them yet.
+An effort was made to sync every animation in vanilla Koikatsu to device actions with an acceptable accuracy, at least where it makes sense. If an animation feels out of sync, and it's not a latency problem, please:
+1. Tell us about it by opening an issue. State the exact pose, pose modifier and excitement state.
+1. If you're a developer, try tweaking the animations.json file. Don't forget to give us a PR.
 
 ## Configuration
+<img src="https://user-images.githubusercontent.com/76826783/126218961-e75500a1-bff4-4ac5-aa52-80f435461a8b.jpg">
 In Plugin Settings > ButtPlugin, you can set the following parameters:
-* The maximum speed your device is capable of ("Maximum strokes per minute"). Based on this value, ButtPlugin will slow down H scene animations if necessary, to keep the immersion. (You'll still be in control of speed, but it will be relative to how fast your toy can go.) The part right before climax will also be slowed down. Keep in mind that the plugin never goes below 70% stroke length, so choose a number your device can reliably maintain at that setting.
-* The latency between your display and your device. Set a negative value if your device is faster than your display. (No way to calibrate, you have to "experiment".)
-* The Buttplug server address (usually localhost:12345).
+
+### Device List
+* **Connect:** Reconnect to the Buttplug server.
+* **Scan:** Scan for devices.
+* **Threesome role:** Which girl the device is assigned to in a threesome. This also affects other scenes - if a device is assigned to second girl, it will not be activated in Standard scenes.
+
```

Comment:
might want to also indicate that currently the columns for Stroker and Vibrator are simply indicators that the device has this functionality and aren't individual toggles to enable/disable the functionality.

Reply:
Done.